server/container_remove.go

```diff
@@ -22,6 +23,12 @@ func (s *Server) RemoveContainer(ctx context.Context, req *types.RemoveContainer
 		return err
 	}
 
+	if s.config.EventedPLEG {
+		if err := s.Runtime().UpdateContainerStatus(ctx, c); err != nil {
+			return fmt.Errorf("failed to update container status %s: %v", req.ContainerId, err)
```

Comment:
```suggestion
			return fmt.Errorf("failed to update container status %s: %w", req.ContainerId, err)
```

also for the other occurrences.

Reply:
Addressed